db/migrate/20230803180807_add_use_year_to_opening_hours.rb

```diff
@@ -0,0 +1,5 @@
+class AddUseYearToOpeningHours < ActiveRecord::Migration[5.2]
+  def change
+    add_column :opening_hours, :use_year, :integer
```

Comment:
how come that you have created with `integer`? what is this decision based on? please update it to `boolean`.

Reply:
I misunderstood something and thought it was about specifying a year, similar to how 1970 is used

Reply:
ah i understand! all right, thanks for the changes.